### PR TITLE
[1069] ucas contact details api spec

### PR DESCRIPTION
**Pull request by misaka**
https://trello.com/c/Flx0PnGG/1069-spec-api-for-ucas-contact-details

### Context

We need to spec out how we serve out contact detail information to UCAS in the API.

### Changes proposed in this pull request

Add contacts detail information.

### Guidance to review

I couldn't help myself and added a brief description to the top, and a table of contents.

This PR is draft because we may be adding another contact type to the contacts for existing contact information which is being imported directly into the provider table.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -373,12 +424,45 @@ endpoint) if any of these are true:
...
     "recruitment_cycle": "2019",
+    "g12_contact_address": "info@asmescitt.education.uk",
     "campuses": [
       {
         "campus_code": "",
         "name": "Main Site",
         "region_code": "01"
       }
+    ],
+    "contacts": [
```
Comment (defong) [inline docs/api.md]: these ain't actual people and contact right?

File: docs/api.md [hunk 2]
```diff
@@ -373,12 +424,45 @@ endpoint) if any of these are true:
     "region_code": "01",
     "scheme_member": "Y",
     "recruitment_cycle": "2019",
+    "g12_contact_address": "info@asmescitt.education.uk",
```
Comment (defong) [inline docs/api.md]: these ain't actual people and contact right?

File: docs/api.md (hunk 1 above)
Comment (misaka) [reply docs/api.md]: No they aren't, I mean, not that I'm aware. Guess I could double-check.

Comment (misaka) [reply docs/api.md]: I've made these obviously fake now.

File: docs/api.md (hunk 2 above)
Comment (misaka) [reply docs/api.md]: I've made these obviously fake now.

File: docs/api.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
 
+This documents the API that serves out post graduate teacher training course,
+subject and training provider information. The version of the API documented
+here feeds the UCAS Apply system.
+
+# Table of Contents
+
+* [Authorisation](#authorisation)
+* [Retrieving Records](#retrieving-records)
+  * [Populating an empty system](#populating-an-empty-system)
```
Comment (dankmitchell) [inline docs/api.md]: Indentation looks broken...

Comment (tvararu) [reply docs/api.md]: I believe it's intentional:

Screenshot 2019-03-18 at 10 29 31

**Comment by misaka** [comment]
[Spec API for UCAS contact details](https://trello.com/c/Flx0PnGG/1069-spec-api-for-ucas-contact-details)

**Pull request merged**